src/components/Article_Card.jsx

```diff
@@ -0,0 +1,41 @@
+import '../styles/ArticleCard.css'
+import { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+const Article = () => {
+    const { article_id } = useParams();
+    const [article, setArticle] = useState(null);
+    const [loading, setLoading] = useState(true);
+
+    useEffect(() => {
+        const fetchArticle = async () => {
+            try {
+                const response = await fetch(`http://localhost:9090/api/articles/${article_id}`);
```

Comment:
using local hosted api will work on your machine but won't work on others. It has to be the api that already deployed. Please change it to your backend api on render